lib/datadog/tracing/contrib/propagation/sql_comment.rb

```diff
@@ -21,7 +21,7 @@ def self.prepend_comment(sql, span_op, trace_op, mode)
             return sql unless mode.enabled?
 
             tags = {
-              Ext::KEY_DATABASE_SERVICE => span_op.service,
+              Ext::KEY_DATABASE_SERVICE => span_op.get_tag('peer.service') || span_op.service,
```

Comment:
Is `'peer.service'` already a constant somewhere, if so we we should reference that constant instead of using an inline string.